gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -27,8 +27,21 @@ local serverColsVar = GetConVar("ttt_bem_sv_cols")
 local serverRowsVar = GetConVar("ttt_bem_sv_rows")
 local serverSizeVar = GetConVar("ttt_bem_sv_size")
 
--- add favorites DB functions
-include("favorites_db.lua")
+-- setup database
+if not sql.TableExists("ttt_bem_fav") then
+	sql.Query("CREATE TABLE ttt_bem_fav (guid TEXT, role TEXT, weapon_id TEXT)")
+end
+
+local upQueryV1 = "CREATE TABLE ttt_equip_fav (guid TEXT, role TEXT, weapon_id TEXT, PRIMARY KEY (guid, role, weapon_id)); " ..
+					"INSERT INTO ttt_equip_fav SELECT * FROM ttt_bem_fav; " ..
+					"DROP TABLE ttt_bem_fav;"
```

Comment:
I'd remove everything that has something todo with `bem`. Just rename it to `shop`

Reply:
that is out of scope for this PR. Also this part is needed as we don't want to throw away already existing data.

What could be discussed is making this migration irreversible by not specifying a "down" query (irreversible migrations will still need to get handled in the migration library and also documented thoroughly)